app/views/admin/cfps/_tracks_cfp.html.haml

```diff
@@ -7,6 +7,10 @@
 %dd#end_date
   = @cfp.end_date.strftime('%A, %B %-d. %Y')
 %dt
+  Description:
+%dd#description
+  = @cfp.description  
```

Comment:
Why isn't this in markdown as well?

Reply:
I think it was missed out by mistake. I will make the changes right away. Thanks for your help.

Reply:
What I mean is why we don't show the description formatted (as per the markdown format the user has used).

Please add relevant screenshots of your changes so that we can see this more clearly.

Reply:
![screenshot from 2017-09-08 17 20 05](https://user-images.githubusercontent.com/22616017/30210553-b56ac460-94ba-11e7-997a-4fb81f7df481.png)
This is how the description text looks as of now.

Reply:
![screenshot from 2017-09-08 17 19 57](https://user-images.githubusercontent.com/22616017/30210653-fdd569da-94ba-11e7-9afa-345977aaa4f1.png)
This is the update form for cfp. Here decription contains markdown format link

Reply:
@ViditChitkara this is the cfps#index view, my comment was about the cfps#show view

Reply:
![screenshot from 2017-09-08 17 36 37](https://user-images.githubusercontent.com/22616017/30210926-4dc433e4-94bc-11e7-9484-39986b2f4504.png)
Thanks @differentreality , I have made the necessary changes.